history-of-quakers/modernized/12-chapter-10.adoc

```diff
@@ -1147,18 +1266,23 @@ and to have a long discourse with him, which soon after he published in print.
 Being admitted into the king`'s presence,
 he gave him a relation of the state of his friends, and said,
 
+[quote]
+____
 Since the Lord has called us, and gathered us to be a people, to walk in his fear,
 and in his truth,
 we have always suffered and been persecuted by the powers that have ruled,
 and been made a prey of, for departing from iniquity;
 and when the breach of no just law could be charged against us,
 then they made laws on purpose to ensnare us;
 and so our sufferings were unjustly continued.
+____
 
-King: It is true, those who have ruled over you have been cruel,
+[.discourse-part]
```

Comment:
So I used [.small-break]'s here several times to separate out some of the text from the discourse. I wasn't sure if should use that or [.offset], but I thought it looked a little better.

Reply:
Ok, yeah, it looks nice in the PDF, let's go with what you did, I like it. In cases like this deciding between offset and small break is tough, and the distinction is not super important.  but I think of offset as being a bit more of a "intrusion" or "interuption of the narrator" often (but not always), so I think I probably would have reached for small-break's here as well.